official/resnet/keras/resnet_model.py

```diff
@@ -0,0 +1,468 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""ResNet50 model for Keras adapted from tf.keras.applications.ResNet50.
```

Comment:
I did not take a very detailed look at this file (and confirm that it is the exact same model as the non-Keras version). Let me know if I should be doing that, otherwise LGTM :)

That said, worth clarifying here how this differs from tf.keras.applications.ResNet50.